### PR TITLE
Reqs for app

### PR DESCRIPTION
**Pull request by abibat-adesina**
I included local deployment paragraph

### CODE REVIEW

File: docs/mkdocs.yml
Comment (wrnu) [inline docs/mkdocs.yml:1]: This file should be removed, it is not needed, there is a mkdoc.yml in the root that should be used. Make sure to add this new page to that file.

File: docs/package-lock.json
Comment (wrnu) [inline docs/package-lock.json:1]: This file should be removed

File: docs/package.json
Comment (wrnu) [inline docs/package.json:1]: this file should be removed

File: .DS_Store
Comment (wrnu) [inline .DS_Store:1]: This file should be removed, there is one in the root that can be removed as well.

Please add a .gitignore file to the root and put `.DS_Store` in it

File: docs/requirements-for-building-your-application.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Requirements for building your application
+
+Last updated: **November 27, 2023**
```
Comment (wrnu) [inline docs/requirements-for-building-your-application.md:3]: This isn't so important but we can update the date :)

**Review by wrnu — CHANGES_REQUESTED** [review]
The content of the document is great! Thanks @abibat-adesina .

A have left a few comments for a few changes that are needed.

Also this branch is behind the `dev` branch, please pull in the latest changes

File: docs/mkdocs.yml
Comment (abibat-adesina) [reply docs/mkdocs.yml:1]: How do I add this new page to mkdoc.yml in the root

Comment (abibat-adesina) [reply docs/mkdocs.yml:1]: I have seen how to add the new page to mkdoc.yml in the root

**Comment by abibat-adesina** [comment]
All changes made

**Review by wrnu — APPROVED** [review]
Looks good :+1:

**Review by Pilargit12 — CHANGES_REQUESTED** [review]
Left a comment in the new PR push - line 47-53 to be addressed as per your expertise, however I believe for the reader it would be helpful to provide further detail. Thanks!

**Comment by abibat-adesina** [comment]
I added more info that might be helpful to the reader

File: docs/requirements-for-building-your-application.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+* Obtain an AWS credential by visiting our AWS login page following the steps below and as shown in the image below:
+
+<!-- Please for accessibility purposes describe each step to take such as AWS login page URL and steps to arrive to click on the button credential 
+
+1. Login to [AWS login page](URL) 
+2. Go to ......
+3. Click..... 
+
+  -->
```
Comment (Pilargit12) [inline docs/requirements-for-building-your-application.md:58]: This is the part that needs last revision @abibat-adesina  - thank you so much!

**Review by abibat-adesina — COMMENTED** [review]
@wrnu @Pilargit12 
Related: bcgov/cloud-pathfinder#2700

**Review by wrnu — APPROVED** [review]

**Review by Pilargit12 — APPROVED** [review]
Hi @abibat-adesina - new pull contains just a few corrections but should be good to go! Thanks!

**Pull request merged**